### DISCUSSION TITLE
elog sync failed："data":{"status":404,"message":"Not Found"},"status":404,

### DISCUSSION THREAD

**Discussion by zhongfeiorz** [LetTTGACO/elog#57] [Q&A] [answered]
elog 同步语雀文档失败，获取文档列表，返回为空，大家有遇到这个问题吗
使用 yuque-hexo 也是这个错误

> elog sync -e ../.elog.env
环境变量  已指定读取env文件为：/home/zf/work/blog/.elog.env
全量更新  未获取到缓存，将全量更新文档
正在获取文档列表，请稍等...
{"data":{"status":404,"message":"Not Found"},"status":404,"headers":{"date":"Mon, 15 Jan 2024 15:51:29 GMT","content-type":"application/json; charset=utf-8","content-length":"36","connection":"keep-alive","set-cookie":

**Comment by zhongfeiorz** [comment, marked as answer]
**原因已找到**：`.elog.env`  文件中 `YUQUE_LOGIN `  配置错了

**语雀个人路径**。访问[工作台](https://www.yuque.com/dashboard) => 账户设置 => 账户设置 => 个人路径，设置语雀的简易的个人路径（建议），拿到个人路径。

**注意**： 比如个人路径为 https://www.yuque.com/abcd ，`YUQUE_LOGIN ` 应设置为  `abcd `，而不是 https://www.yuque.com/abcd

**PS**：一定要仔细看[文档](https://elog.1874.cool/notion/gvnxobqogetukays#%E8%AF%AD%E9%9B%80) 😂

**Reply by LetTTGACO** [reply]
好的，不过也提醒我了，我已经在相关主流程加上了一些校验和友好日志提示
